cli.py

```diff
@@ -4,25 +4,19 @@
 from therapy.etl import ChEMBL, Wikidata, DrugBank, NCIt
 from therapy.schemas import SourceName
 from timeit import default_timer as timer
-from therapy.database import Database, DYNAMODBCLIENT, THERAPIES_TABLE, \
-    METADATA_TABLE
+from therapy.database import Database
 from boto3.dynamodb.conditions import Key
 
 
 class CLI:
     """Class for updating the normalizer database via Click"""
 
     @click.command()
-    @click.option(
-        '--all',
-        is_flag=True,
-        help='Update all normalizer databases.'
-    )
```

Comment:
Why are we removing this flag? We should keep it so that users don't have to type in each normalizer.